Userland/Libraries/LibGUI/WordWrapMode.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021-2021, Vinicius Sugimoto <vtmsugimoto@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Optional.h>
+#include <AK/StringView.h>
+
+namespace GUI {
+
+#define GUI_ENUMERATE_WORD_WRAP_MODES(M) \
+    M(Word)                              \
+    M(Anywhere)
+
+enum class WordWrapMode {
+#define __ENUMERATE(x) x,
+    GUI_ENUMERATE_WORD_WRAP_MODES(__ENUMERATE)
+#undef __ENUMERATE
+};
+
+inline Optional<WordWrapMode> word_wrap_mode_from_string(const StringView& string)
```

Comment:
Simple `inline`s are often ignored by the compiler and therefore not useful for things we want to inline all the time. Let's use the `ALWAYS_INLINE` macro. Also, east const.
```suggestion
ALWAYS_INLINE Optional<WordWrapMode> word_wrap_mode_from_string(StringView const& string)
```

Reply:
I wrote this file based on <LibGfx/TextAlignment.h>, which also uses simple `inlines`, so maybe those should be changed too

Reply:
That's for another PR :yakstack: